### PR TITLE
Lizard fixes

### PR DESCRIPTION
**Pull request by CollenN**
Fixes lizard sprites again, and also re-adds several of the old lizard horns to the game. Also, the tiger stripe body markings have been made lighter in order to accommodate the new sprites.

Also also, it's now possible to change which icon is used by sprite accessories.

### CODE REVIEW

**Comment by Profakos** [comment]
What was the fix though? If the fact that they were too dark, please do not change that because every single mutant bodypart is coloured to match its darkness value.

**Comment by CollenN** [comment]
Nah, the bodyparts look just fine now.

**Comment by Profakos** [comment]
You mean you edited the bodyparts too? That is even worse, now when a new bodypart is added we have to keep editing them forever.

**Comment by Profakos** [comment]
Also please don't commit in the unrelated merge conflict files (it is probably caused by line endings, we need to fix those)

**Comment by Profakos** [comment]
Other review things: you were supposed to put the oldhorns into a new file so the mutant bodyparts dmi won't get edited :P

File: code/modules/mob/new_player/sayu_species_accessories.dm [hunk 1]
```diff
@@ -2,75 +2,49 @@
 // Species accessories //
 /////////////////////////
 
-/datum/sprite_accessory/species
+/datum/sprite_accessory/horns/sayu
 	icon = 'icons/mob/spec_accessory.dmi'
```
Comment (Profakos) [inline code/modules/mob/new_player/sayu_species_accessories.dm:6]: this will show up in the hornlist

**Comment by Profakos** [comment]
Making the bodyparts use a nonhardcoded file is a very good change :+1:

File: code/modules/mob/new_player/sayu_species_accessories.dm (hunk 1 above)
Comment (CollenN) [inline code/modules/mob/new_player/sayu_species_accessories.dm:6]: That's because they're horns

Comment (Profakos) [inline code/modules/mob/new_player/sayu_species_accessories.dm:6]: I mean this is the base abstract horn object with no associated icon_state, or name

**Pull request closed without merging**